pom.xml

```diff
@@ -206,17 +203,37 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>1.10.19</version>
+            </dependency>
+            <dependency>
+                <groupId>org.objenesis</groupId>
+                <artifactId>objenesis</artifactId>
+                <version>2.6</version>
+            </dependency>
+            <dependency>
+                <groupId>org.hamcrest</groupId>
+                <artifactId>hamcrest-core</artifactId>
+                <version>1.3</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
```

Comment:
shy do we need it?

Reply:
We don't need it, but some transient dependencies would like it managed. Management of dependencies is also ensuring that all transient dependencies are correctly versioned. If you look into the dependency tree you will see duplicate unmanaged transient dependencies (different versions). Defining the specific version enforces the build against a stable base (else the maven reactor will 'choose' the one nearest the top of the tree, which might not be the correct version)

Reply:
We didn't have complain about guava as it was removed time ago so I would like to not add dependencies and maintain it only because other libs use it